test/helpers/kubeclient_helper.rb

```diff
@@ -0,0 +1,21 @@
+module KubeclientHelper
+  def kubeclient
+    @kubeclient ||= begin
+      config = Kubeclient::Config.read(ENV["KUBECONFIG"])
+      unless config.contexts.include?("minikube")
+        raise "`minikube` context must be configured in your KUBECONFIG (#{ENV["KUBECONFIG"]})"
```

Comment:
Absolutely, this should have been `raise` rather than `puts`. No idea why I used `puts` 🙈